src/Services/DocumentSign/SignRequest.php

```diff
@@ -50,18 +49,12 @@ public function signer(string $email, string $firstname = '', string $lastname =
         return $this;
     }
 
-    /**
-     * Creates a Document
-     * @param UploadedFile $file
-     * @param string|null $webhook
-     * @return array
-     */
-    public function create(UploadedFile $file, string $webhook = null): array
+    private function createRequest(string $file, $name, $webhook = null)
```

Comment:
missing doc block, and name/webhook should be typehinted

Reply:
What type should webhook be?